python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1881,52 +1932,53 @@ async def set_atspectrograph_corrections(self, azimuth, elevation):
                 focusOffset=_offset_value, pointingOffsets=_pointing_offsets
             )
 
-            try:
-                # Don't need a tolerance since these values only get set if
-                # they meet a tolerance already
-                if abs(_offset_value):
-                    self.log.debug("Applying focus correction with hexapod")
-                    await self.set_hexapod(azimuth, elevation)
-
-                if np.max(abs(_pointing_offsets)):
-                    self.log.debug(
-                        "Applying pointing correction _pointing_offsets"
-                        f" = {_pointing_offsets} with atcs"
-                    )
-                    # apply offsets relative to what is already there
-                    await self.atcs.offset_xy(
-                        _pointing_offsets[0],
-                        _pointing_offsets[1],
-                        relative=True,
-                        persistent=True,
-                    )
-                    await asyncio.sleep(0)
-                    # update accounting and remove the already applied offsets
-                    self.pointing_offsets_per_category["total"] += _pointing_offsets
-                    self.pointing_offsets_yet_to_be_applied -= _pointing_offsets
+            # Apply focus correction if above tolerance
+            if abs(_offset_value):
+                self.log.debug(
+                    f"Applying focus correction {_offset_value}" " with hexapod"
+                )
+                await self.set_hexapod(azimuth, elevation)
 
-                    self.log.debug(
-                        "new value of pointing_offsets_per_category['total'] is"
-                        f" {self.pointing_offsets_per_category['total']}"
-                    )
-                    self.evt_pointingOffsetSummary.set_put(
-                        total=self.pointing_offsets_per_category["total"],
-                        filter=self.pointing_offsets_per_category["filter"],
-                        disperser=self.pointing_offsets_per_category["disperser"],
-                    )
+            # Apply pointing correction if above tolerance
+            if np.max(abs(_pointing_offsets)):
```

Comment:
Missing a comparison value here?

Reply:
No, it just needs to be non-zero. The tolerance check is done on line 1904

Reply:
so, in this case you probably want to make it `np.max(abs(_pointing_offsets)) > 0.` to make your intention explicit. 

Although, I will say that doing a float comparison to zero may be unsafe. Because, unless you are initializing `_pointing_offsets = 0.` you may have conditions where a floating point operation that should equal to zero does not due to floating point arithmetics.

Reply:
ok, i added the `>` and a comment saying the comparison to zero should be a non-issue as the `_pointing_offsets` is set to zero explicitly and only modified after the tolerance evaluation.